reference/import_export/product-export.rst

```diff
@@ -40,67 +34,46 @@ We can also count a single step named ``export``.
+            - '@pim_connector.writer.file.csv_product' # Writer
+            - 10 # Batch size
 
 An ``ItemStep`` always contains 3 elements, a ``Akeneo\Bundle\BatchBundle\Item\ItemReaderInterface``, a ``Akeneo\Bundle\BatchBundle\Item\ItemProcessorInterface`` and a ``Akeneo\Bundle\BatchBundle\Item\ItemWriterInterface``.
 
-We provide here specific implementations for these elements, the services are declared with aliases ``pim_base_connector.reader.doctrine.product``, ``pim_base_connector.processor.product_to_flat_array``, ``pim_connector.writer.file.csv_product``.
+We provide here specific implementations for these elements, the services are declared with aliases ``pim_connector.reader.database.product``, ``pim_connector.processor.normalization.product``, ``pim_connector.writer.file.csv_product``.
 
 Product Reader
 --------------
 
 This element reads products from database and returns objects one by one.
 
-The service is defined in ``src\Pim\Bundle\BaseConnectorBundle\Resources\config\readers.yml``.
+The service is defined in ``src/Pim/Bundle/ConnectorBundle/Resources/config/readers.yml``.
 
 .. code-block:: yaml
 
     services:
-        pim_base_connector.reader.doctrine.product:
-            class: %pim_base_connector.reader.doctrine.product.class%
+        pim_connector.reader.database.product:
+            class: '%pim_connector.reader.database.product.class%'
             arguments:
-                - '@pim_catalog.repository.product'
-                - '@pim_catalog.manager.channel'
+                - '@pim_catalog.query.product_query_builder_factory'
+                - '@pim_catalog.repository.channel'
                 - '@pim_catalog.manager.completeness'
-                - '@pim_transform.converter.metric'
-                - '@pim_catalog.object_manager.product'
-
-You can notice that the class parameter is not defined in the same file.
-
-Depending on how you store your product data, a specific file will be loaded (orm.yml or mongodb-odm.yml).
-
-For ORM, the parameter is defined in ``src\Pim\Bundle\BaseConnectorBundle\Resources\config\storage_driver\doctrine\orm.yml``.
-
-.. code-block:: yaml
-
-    parameters:
-        pim_base_connector.reader.doctrine.product.class: Pim\Bundle\BaseConnectorBundle\Reader\Doctrine\ORMProductReader
-
-For MongoDBODM, the parameter is defined in ``src\Pim\Bundle\BaseConnectorBundle\Resources\config\storage_driver\doctrine\mongodb-odm.yml``.
-
-.. code-block:: yaml
-
-    parameters:
-        pim_base_connector.reader.doctrine.product.class: Pim\Bundle\BaseConnectorBundle\Reader\Doctrine\ODMProductReader
-
-The reader will only return products that are complete for the selected channel, classified in a category and enabled.
-
-.. note::
+                - '@pim_catalog.converter.metric'
+                - true
 
     To know more about how we load different configuration depending on the storage driver you can take a look at ``Pim\Bundle\CatalogBundle\DependencyInjection\PimCatalogExtension``
```

Comment:
To know more about how we load different configuration depending on the storage driver ***, ***